clients/src/main/java/org/apache/kafka/clients/telemetry/DefaultClientTelemetry.java

```diff
@@ -385,8 +387,12 @@ private int getPushIntervalMs(final GetTelemetrySubscriptionsResponseData data)
     }
 
     @Override
-    public void pushTelemetrySucceeded(PushTelemetryResponseData data) {
+    public void pushTelemetryReceived(PushTelemetryResponseData data) {
         log.debug("Successfully pushed telemetry; response: {}", data);
```

Comment:
The 'successfully pushed' message should only be logged if `data.errorCode() == Errors.NONE.code()`. Maybe consider putting that in an `else` block?